module/os/linux/zfs/zpl_super.c

```diff
@@ -29,7 +29,10 @@
 #include <sys/zfs_vnops.h>
 #include <sys/zfs_ctldir.h>
 #include <sys/zpl.h>
+#include <sys/dmu_objset.h>
+#include <sys/dsl_dir.h>
 
+int zfs_forced_export_unmount_enabled = 0;
```

Comment:
This needs to be added to the `man/man4/zfs.4` man page.

Reply:
Done. (It was already, but the name was missing the`_enabled` word.